src/Symfony/Component/Process/Shells/ShellCommandExecutor.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Symfony\Component\Process\Shells;
+
+use Symfony\Component\Process\Contracts\CommandExecutorInterface;
+
+/**
+ * ShellCommandExecutor default shell executor.
+ *
+ * @author Pululu Kinanga Andre <pululuandre@gmail.com>
+ */
+class ShellCommandExecutor implements CommandExecutorInterface
+{
+    /**
+     * Executes a shell command.
+     *
+     * @param string $command The shell command to execute.
+     * @return string|null The output of the command.
+     */
+    public function execute(string $command): ?string
```

Comment:
this implementation should not exist. It is the Process class without any of the features making the Process class work fine cross-plateform.

Reply:
Fixe